### DISCUSSION TITLE
Attention for phishing mail!

### DISCUSSION THREAD

**Discussion by Edd-Gao** [community/community#110631] [Other Feature Feedback, Questions, & Ideas]
I received this phishing mail saying there is a security breach in Github. Of course there isn't. This link asks for tons of accesses to my account. Luckly I'm very clever I don't authorize them. What do they want from my account?!!
![attention-for-phishing-mail-v0-kfd0aszllolc1 png webp](https://github.com/community/community/assets/4394777/c34d1562-724e-4db1-b33e-a7afc65fc6ea)
![attention-for-phishing-mail-v0-4suv56cylolc1 png webp](https://github.com/community/community/assets/4394777/814be17f-b68a-441b-a599-adef45874f3e)

**Comment by Akash1134** [comment]
Hi @Edd-Gao , 

We understand the inconvenience caused by these notifications. Our teams are currently working on addressing these unsolicited phishing notifications. We want to remind our users to continue to use [our abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) to raise any abusive or suspicious activity.

We would like to bring to our users’ attention to the following:

- Please do not click any links or reply to these notifications. Please [report them](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam).
- [Authorizing an OAuth app](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/authorizing-oauth-apps) can expose your GitHub account and data to a third party.
- GitHub recommends you periodically [review your authorized OAuth apps](https://docs.github.com/en/apps/oauth-apps/using-oauth-apps/reviewing-your-authorized-oauth-apps).
- This spam activity targets and abuses GitHub’s mention and notification functionality.
- This is a phishing campaign and is not the result of a compromise of GitHub or its systems.
